# Botany2022NMGWorkshop

Title: de novo genome assembly and annotation with an emphasis on phylogenetic and population genetic studies

Presenters: Gisel De La Cerda, Jacob Landis, Fay-Wei Li, and Susan Strickler

Length: 8 am ADT - 5 pm ADT

The costs of generating a draft genome for a study organism of choice are declining each year and more tools are becoming available to help in the endeavor. However, generating the appropriate sequence data is often not a trivial matter and currently, there are many options when it comes to analysis pipelines. The focus of this workshop is to work through the necessary steps for genome assembly and annotation when a closely related reference genome is not available. The workshop will cover assembling genomes using Illumina, Oxford Nanopore, and hybrid assemblers; as well as methods for extracting high-quality DNA suitable for ONT sequencing and library preparation, especially in species that possess secondary compounds which can be especially problematic for nanopore sequencing. Downstream applications/analyses that can be performed after assembly will also be covered while highlighting that a perfect assembly is not necessary to answer many evolutionary questions. Using the CyVerse platform, participants will get hands-on practice with all the necessary steps using either the supplied test data or their own data if previously generated.

Topics and Layout
Prerequisites and baseline skills:
Unix command-line skills (we will review these at the start)
Familiarly with some basic programming
CyVerse account
CoGe account

Syllabus

Introductions and overview genome sequencing and assembly 8 am (30 min) - Suzy & Gisel

Setting hardware/software baseline skills 8:30 (45 min) - Gisel

Generate suitable DNA for different platforms: tips, tricks, and experimental design 9:15 am (30 min) - Fay-Wei

Break 9:45 am (45 min)

Sequencing data and genome assembly options 10:30 am (1.5 h) - Jacob

Break for lunch 12:00 pm (1 h)

Genome assembly wrap up 1:00 (30 min) - Jacob

Genome annotation 1:30 (1.5 h) - Suzy

Break 3:00 pm (30 min)

Downstream evolution analyses: genome management, genome visualization, comparative genomics, reference-based SNP calling, transcriptomics, and phylogenomics 3:30 (1 h) -  Jacob

Wrap-up and discussion 4:30 - All
